configs/components/openssl-1.1.1.rb

```diff
@@ -1,6 +1,6 @@
 component 'openssl' do |pkg, settings, platform|
-  pkg.version '1.1.1v'
-  pkg.sha256sum 'd6697e2871e77238460402e9362d47d18382b15ef9f246aba6c7bd780d38a6b0'
+  pkg.version '1.1.1w'
```

Comment:
Where are we getting this `1.1.1w` package?
I thought `1.1.1v` was the last version?